tmb-user-mgt/tmb-userdata-mock/src/main/java/de/interseroh/tmb/user/client/UserInformationServiceImpl.java

```diff
@@ -0,0 +1,99 @@
+package de.interseroh.tmb.user.client;
+
+import com.google.gwt.core.client.Callback;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.user.client.Cookies;
+import org.fusesource.restygwt.client.MethodCallback;
+import org.gwtbootstrap3.client.ui.AnchorButton;
+import org.gwtbootstrap3.client.ui.base.ComplexWidget;
+import org.gwtbootstrap3.client.ui.constants.ButtonType;
+
+import java.util.logging.Logger;
+
+
+/**
+ * Mock User Service
+ */
+public class UserInformationServiceImpl implements UserInformationService{
+
+    private static final Logger logger = Logger
+            .getLogger(UserInformationServiceImpl.class.getName());
+
+    public UserInformationServiceImpl(String gatewayLocation, String userInfoUrl, String cookiePath, String logoutUrl) {
+        logger.info("USER INFORMATION MOCK HAS BEEN CONFIGURED WITH "
+                +(gatewayLocation==null || gatewayLocation.trim().isEmpty()? " NO GATEWAY ":gatewayLocation)
+                + "\n AND "
+                +(userInfoUrl == null || userInfoUrl.trim().isEmpty() ? " NO USER INFO URL ": userInfoUrl)
+                + "\n AND "
+                + (cookiePath == null | cookiePath.trim().isEmpty()? " NO" : cookiePath)+" COOKIE PATH"
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Correct this "|" to "||". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2178)